### PR TITLE
CPDP-1800: [POA]Single Disposal Property Details message typos fixes

### PR DESCRIPTION
**Pull request by rambabu-posa**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -1331,19 +1331,19 @@ property-details.multiple-disposals.guidance.capacitor.p3=Contract exchange usua
 property-details.multiple-disposals.guidance.p4=You’ll also need to upload a document later providing details of the other properties you’re including in this return.
 
 enterPostcode.returns.singleDisposal.title=Find the address of the UK property you disposed of
-enterPostcode.returns.agent.singleDisposal.title=Find the address of the UK property your client disposed of
+enterPostcode.returns.agent.singleDisposal.title=Find the address of the UK property the person disposed of
```
Comment (AndyHWChung) [inline conf/messages:1334]: does this content change apply to all agents or just agents of period of admin people?

Comment (rambabu-posa) [reply conf/messages:1334]: As per story description " The user enters this user journey when they answer 'Who are you completing this return for?' with 'The estate of a person who died before disposing of a property."

I think it applies to "Agent of an Estate" only.

Comment (AndyHWChung) [reply conf/messages:1334]: you need a separate user key to cover this then. I used `.personalRepInPeriodOfAdmin.agent` for mine when I had something similar - the pattern match for the user key will have to then have a case to return that value.

Comment (AndyHWChung) [reply conf/messages:1334]: if it helps this is what I ended up doing: https://github.com/hmrc/cgt-property-disposals-frontend/blob/master/app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/disposaldetails/disposal_fees.scala.html#L63

File: conf/messages [hunk 2]
```diff
@@ -1332,18 +1332,21 @@ property-details.multiple-disposals.guidance.p4=You’ll also need to upload a d
 
 enterPostcode.returns.singleDisposal.title=Find the address of the UK property you disposed of
 enterPostcode.returns.agent.singleDisposal.title=Find the address of the UK property your client disposed of
+enterPostcode.returns.personalRepInPeriodOfAdmin.agent.singleDisposal.title=Find the address of the UK property the person disposed of
```
Comment (AndyHWChung) [inline conf/messages:1335]: does the template return this user key already?

Comment (rambabu-posa) [reply conf/messages:1335]: Nope, Im still working on that one

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/address/enter_postcode.scala.html [hunk 3]
```diff
@@ -65,18 +65,26 @@
   }
 }
 
+@userMsgKey = @{
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/address/enter_postcode.scala.html:68]: something like `titleUserMessageKey` would be more descriptive here - it's hard to tell what the difference is between `userMsgKey` and `userKey`

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**